lms/views/lti/basic_launch.py

```diff
@@ -240,26 +228,20 @@ def legacy_vitalsource_launch(self):
             )
         )
 
-    def _course_copied_launch(self, original_resource_link_id):
+    def _assignment_based_launch(self, resource_link_id):
```

Comment:
Same thing here, everything is "assignment based" here I reckon. What's speceial here is the DB (stored_assignment? ) part.

Reply:
Yeah that's true. I was meaning it as in the DB model, but as a word it applies to everything